src/coreclr/build-test.sh

```diff
@@ -5,6 +5,10 @@ build_test_wrappers()
     if [[ "$__BuildTestWrappers" -ne -0 ]]; then
         echo "${__MsgPrefix}Creating test wrappers..."
 
+        if [[ $__Mono=1 ]]; then
+            export EXCLUDE_MONO_FAILURES="true"
```

Comment:
For better alignment with other conditional sections in issues.targets, could you rather expose a variable named something like RuntimeType with the value "coreclr" or "mono" and change the check in issues.targets to <code>'$(RuntimeType)' == 'mono'</code>?

Reply:
Ok, that makes sense. I will do it that way.